packages/ui-lib/src/Filters/atoms/ToggleButton.tsx

```diff
@@ -23,7 +23,8 @@ const ToggleButton: React.FC<IToggleButton> = ({ options, categoryLabel, selecte
 
   return (
     <FilterButton icon={options[selectedIndex].icon} onClick={() => onToggleCallback(selectedIndex)} {...{design}} {...props}>
-      {`${categoryLabel} : ${options[selectedIndex].text}`}
+      { categoryLabel && `${categoryLabel} : `}
```

Comment:
Can we remove the whitespace between the label and the `:`, so it sits against it, for example: "Layout: Grid"

Reply:
Thanks for catching this one :D!